.travis.yml

```diff
@@ -0,0 +1,9 @@
+dist: xenial
+sudo: required
+before_install:
+    - sudo add-apt-repository -y ppa:fish-shell/release-2
```

Comment:
Do we really need `sudo` here?

Reply:
I took the [config from `fishtape`](https://github.com/fisherman/fishtape/blob/master/.travis.yml)